model/group_syncable.go

```diff
@@ -82,49 +80,6 @@ func (syncable *GroupSyncable) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-func (syncable *GroupSyncable) MarshalJSON() ([]byte, error) {
```

Comment:
This change breaks the database creation. If you try to reset the database and create it again, its broken.

Reply:
Yes I'm looking into the fix.